src/index.js

```diff
@@ -9,6 +9,7 @@ import autoprefixer from 'autoprefixer';
 import cssnano from 'cssnano';
 import { rollup, watch } from 'rollup';
 import builtinModules from 'builtin-modules';
+import resolveFrom from 'resolve-from';
```

Comment:
why do we need this instead of just `require.resolve` and will this break with something like yarn pnp / pnpm?

Reply:
Not much different in theory. "resolve-from" needs node v8.0 while `paths` options of `require.resolve` requires node v8.9.  And "resolve-from" support `.silent` which will lead to a bit nicer code in this case.

This PR works with yarn, I'm using yarn actually.  But I have no idea about pnpm.

BTW there's [an issue in "require-resolve"](https://github.com/sindresorhus/resolve-from/issues/8) which complained that `paths` of `require.resolve` doesn't work. But I never dug deeper about that.

Reply:
Node 8 is EOL so that doesn't matter

You're using yarn 2? Then it's fine to merge